Server/UserManagement/UserManagement.Application/CQRS/CommandHandlers/CreateAreaCommandHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using ModularHouse.Libraries.InternalMessaging.CQRS.Abstractions.Command;
+using ModularHouse.Libraries.InternalMessaging.DomainEvents.Abstractions;
+using ModularHouse.Server.Common.Domain;
+using ModularHouse.Server.Common.Domain.Exceptions;
+using ModularHouse.Server.UserManagement.Application.CQRS.Commands;
+using ModularHouse.Server.UserManagement.Application.MappingExtensions;
+using ModularHouse.Server.UserManagement.Domain.AreaAggregate;
+using ModularHouse.Server.UserManagement.Domain.AreaAggregate.Events;
+
+namespace ModularHouse.Server.UserManagement.Application.CQRS.CommandHandlers;
+
+public class CreateAreaCommandHandler : ICommandHandler<CreateAreaCommand>
+{
+    private readonly IAreaDataSource _dataSource;
+    private readonly IAreaRepository _repository;
+    private readonly IDomainEventBus _domainEventBus;
+
+    public CreateAreaCommandHandler(
+        IAreaDataSource dataSource,
+        IAreaRepository repository,
+        IDomainEventBus domainEventBus)
+    {
+        _dataSource = dataSource;
+        _repository = repository;
+        _domainEventBus = domainEventBus;
+    }
+
+    public async Task Handle(CreateAreaCommand request, CancellationToken cancellationToken)
```

Comment:
I would change: request => command in the CommandHandlers, and request => query in the QueryHandlers imho. But feel free to use your discretion.